feat(codemod): codemods and recipe for migrating Sentry.js from v7.x to v8.x

akash-kumar-dev opened the pull request:
## Sentry v7.x → Sentry  v8.x

### 📚 Description

- **Link to Official Upgrade Guide**: [Link](https://docs.sentry.io/platforms/javascript/migration/v7-to-v8/)

#### Sentry/V8/Removal Severity Enum
- Registry [Link](https://codemod.com/registry/sentry-v8-removal-Severity-Enum)
-  [Codemod Studio Link](https://go.codemod.com/zYCUpF1)

#### Sentry/V8/Removal Sentry.ConfigureScope Method
- Registry [Link](https://codemod.com/registry/sentry-v8-Removal-Sentry-configureScope-method)
- [Codemod Studio Link](https://go.codemod.com/KavKUiF)

#### Sentry/V8/Replace Span Status From Http Code
- Registry [Link](https://codemod.com/registry/sentry-v8-replace-span-status-from-http-code): [Link](https://codemod.com/registry/sentry-v8-replace-span-status-from-http-code)
- [Codemod Studio Link](https://go.codemod.com/yK5xQD9)

#### Sentry/V8/Removal Of AddGlobalEventProcessor
- Registry [Link](https://codemod.com/registry/sentry-v8-removal-of-addGlobalEventProcessor)
- [Codemod Studio Link](https://go.codemod.com/l2FucZz)

#### Sentry/V8/Remove Replay Package And Update Integration
- Registry Link: [Link](https://codemod.com/registry/sentry-v8-remove-replay-package-and-update-integration)
- [Codemod Studio Link](https://go.codemod.com/X9PRS06)

#### Sentry/V8/Removal Of Void From Transport Return Types
- Registry [Link](https://codemod.com/registry/sentry-v8-removal-of-void-from-transport-return-types)
- [Codemod Studio Link](https://go.codemod.com/A6ZiBDm)](https://go.codemod.com/A6ZiBDm)

#### Sentry/V8/Migration Recipe
- Registry [Link](https://codemod.com/registry/sentry-v8-migration-recipe)

mohab-sameh commented:
@akash-kumar-dev We're moving codemod packages to their corresponding owners so it's easier for codemod owners to own and maintain their codemods, and be notified about any issues in their own repos. Do you mind moving this to your own repo and updating the GitHub source URL of the published codemod to point to your github repo?

1. **Create a repo** in your own GitHub account and move the source code of your codemods to that repo.
     - This will drive more traffic to your profile and clarify ownership, increasing visibility and transparency. 
     - You could name the repo something like codemods (or choose a name you prefer).
     - For the README of that repo, you can use a format similar to what's in the [commons](https://github.com/codemod-com/commons/) repo.

2. **Update the git URL** in the codemods’ metadata to your new repo and bump their versions.
3. **Republish the codemods** to ensure they are showing the correct git URL in the registry.
4. Optionally, you can duplicate [this GitHub Action](https://github.com/codemod-com/Commons/blob/main/.github/workflows/main.yaml) for your repo. This way, every time you make a change to one of your codemods and merge the code to the main branch, it will automatically publish the codemod for you.

akash-kumar-dev commented:
Hi @mohab-sameh , 
I've moved the whole repo to my own GitHub, [codemods](https://github.com/akash-kumar-dev/codemods). I also updated the GitHub link from all codemods in the codemod registry. The only thing remaining is to add GitHub actions, which I will do soon. I'm going to update all my other codemods from the registry soon and move them to my own repo as well.

mohab-sameh commented:
Awesome @akash-kumar-dev. Thanks a lot!
